airflow_config/dags/optim_dag.py

```diff
@@ -1,7 +1,7 @@
 from airflow import DAG
```

Comment:
These changes are mine, check them @pchtsp

Reply:
For the `solver` functionality of cornflow (`optim_dag.py`) I think we should just leave one task, because there is nothing to do before or after solving and it increases time.
For other, more complicated, dags (specific to problems) this makes sense since the machines with expensive solvers may be a bottleneck and the pre-processing and post-processing could be done somewhere else without license. We will have to manage nevertheless the data between the machines...

Reply:
Ok, we can simplify this one. Undo my changes and delete the preceding task and do something like that for future dags